charts/aws-ebs-csi-driver/crds/snapshot-controller.yaml

```diff
@@ -0,0 +1 @@
+deploy/kubernetes/cluster/crd_snapshotter.yaml
```

Comment:
It would more sense to move the file here and make the helm chart the "canonical" source for the CRD. Because every other file in the deploy/kubernetes directory is generated from the helm chart by the `make generate-kustomize` rule.

So my suggestion is basically:
1. Move crd_snapshotter.yaml to charts/aws-ebs-csi-driver/crds and maybe rename it?
2. add a line in make generate-kustomize that copies FROM charts/aws-ebs-csi-driver/crds TO deploy/kubernetes/cluster (i.e., the opposite direction as today)

Reply:
sure, I will do this. Give me a couple of hours (I'm stuck in meetings for a while)